diff --git a/kisswm.h b/kisswm.h
index 8e36af6..3daf989 100644
--- a/kisswm.h
+++ b/kisswm.h
@@ -1,7 +1,7 @@
 char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
-static const char dmenufont[]       = "Fira Code Nerd Font:pixelsize=18:antialias=true";
+static const char dmenufont[]       = "Fira Code Nerd Font:pixelsize=28:antialias=true";
 
-static const char *barfont       = "Fira Code Nerd Font:pixelsize=18:antialias=true";
+static const char *barfont       = "Fira Code Nerd Font:pixelsize=28:antialias=true";
 static int barheight = 40;
 
 static const char *term[] = { "st", NULL };
